.github/workflows/refresh-wordpress-major-and-beta.yml

```diff
@@ -39,22 +39,25 @@ jobs:
                   curl -fsSL https://bun.sh/install | bash
             - uses: ./.github/actions/prepare-playground
             - name: 'Recompile WordPress'
+              id: build
               shell: bash
               env:
                   FORCE_REBUILD: ${{ github.event.inputs.force_rebuild }}
               run: PATH="${PATH}:${HOME}/.bun/bin" npx nx bundle-wordpress:major-and-beta playground-wordpress-builds
             - name: Check for uncommitted changes
+              # Prevent commit if the build failed to avoid polluting the commit history
+              if: steps.build.outcome == 'success'
```

Comment:
Is this already the default behavior?

The [documentation for `continue-on-error`](https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#jobsjob_idstepscontinue-on-error) seems to imply that the steps default to stopping on error because you have to set that option to "true" to cause next steps to run anyway.